files/base.rb

```diff
@@ -334,6 +334,11 @@ def default_pager_channel
     "#{team_name}-pages"
   end
 
+  def default_pager
+    return [] if handler_settings['use_default_pager'] == false
```

Comment:
Comparing something to a boolean constant in a condition is a red flag.

Is this same as:

```
return default_pager_channel if handler_settings['use_default_pager']
[]
```

Reply:
unfortunately that's not the same. 

when the configuration does not exist we need the previous behavior which was to supply the team-pager  channel.   ie we care that it's false vs  nil.   

I welcome any suggestions though.   I wasn't 100% happy with it.

Reply:
I did this.   seems better.  wdyt? 
```
    def default_pager
      return default_pager_channel if handler_settings.fetch('use_default_pager',true)
      []  
    end 
```